### PR TITLE
.circleci/config.yml: cache dependencies

### PR DESCRIPTION
**Pull request by simonpasquier**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -21,6 +22,12 @@ jobs:
 
     steps:
     - prometheus/setup_environment
+    - go/load-cache:
+        key: v1
+    - restore_cache:
+        keys:
+        - v1-npm-deps-{{ checksum "web/ui/react-app/yarn.lock" }}
+        - v1-npm-deps-
```
Comment (juliusv) [inline .circleci/config.yml]: Do we need the fallback, since `yarn.lock` should always be there?

**Comment by juliusv** [comment]
:+1: Either way

File: .circleci/config.yml (hunk 1 above)
Comment (SuperQ) [reply .circleci/config.yml]: +1, I'm not sure exactly how this work.

Comment (juliusv) [reply .circleci/config.yml]: Ah ok, makes sense. :+1:

**Pull request merged**